### PR TITLE
F24/justin/add user management and admin pages

### PR DESCRIPTION
**Pull request by JustinScitech**
## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)

## Implementation description
* Created a skeleton user management page that only admins and behaviourists can access. This can be found at the following route: admin/users

## Steps to test
1. Just test using roles as an admin or behavourist if you can go to the route: /admin/users

## What should reviewers focus on?
* Just focus on accessing using the different roles
## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -131,6 +132,18 @@ const App = (): React.ReactElement => {
                 component={ProfilePage}
                 allowedRoles={AuthConstants.ALL_ROLES}
               />
+              <PrivateRoute
+                exact
+                path={Routes.ADMIN_PAGE}
+                component={Default}
```
Comment (jerry-cheng5) [inline frontend/src/App.tsx:138]: should you be passing in the AdminPage component here?

**Review by jerry-cheng5 — COMMENTED** [review]
looks good for the most part, please keep the admin page. It may be converted into an admin dashboard in the future!

File: frontend/src/App.tsx (hunk 1 above)
Comment (JustinScitech) [reply frontend/src/App.tsx:138]: Oh yep mb

**Review by jerry-cheng5 — APPROVED** [review]
Looks good!

**Pull request merged**